CODEOWNERS

```diff
@@ -1 +1,12 @@
-*       @elatt @alexromanyuk @andrius-senulis
+*                                                        @datarobot/machine-learning-development
+/datarobot_provider/operators/ai_catalog.py              @datarobot/mldata
+/datarobot_provider/operators/connections.py             @datarobot/mldata
+/datarobot_provider/operators/feature_discovery.py       @datarobot/trust-explainable-ai
+/datarobot_provider/operators/mlops.py                   @datarobot/model-management
+/datarobot_provider/operators/model_insights.py          @datarobot/trust-explainable-ai
+/datarobot_provider/operators/model_package.py           @datarobot/model-management
+/datarobot_provider/operators/model_predictions.py       @datarobot/predictions
+/datarobot_provider/operators/monitoring.py              @datarobot/model-management
```

Comment:
FYI we will merge `mlops.py` and `monitoring.py` into `deployment.py`, so these two files will be deleted soon